models.js

```diff
@@ -138,3 +139,10 @@ exports.fetchUsers = () => {
     return response.rows;
   });
 };
+
```

Comment:
You shouldn't technically need a model for this endpoint. Models are usually reserved for direct database interaction. It's not the end of the world though 🚁